e2e/features/address-validation.feature

```diff
@@ -22,3 +22,16 @@ Feature: Address details are validated
 
     When I select No for the Does the appellant have a fixed address? field
     When I should see the text We will use the address of your legal practice.
+
+  @create-case @alternate @RIA-690
+  Scenario: Postcode without any addresses
+
+    Given I am on the Your client's address page
+    When I select Yes for the Does the appellant have a fixed address? field
+    Then I should not see the text No address found
+
+    When I type N01 RAL for the Enter a UK postcode field
+    And I click the Find address button
+
+    Then I should see the text No address found
```

Comment:
@fantasticjamieburns is this functionality that CCD gives us for free? If it is I don't think we need to test it our side as well - that should be covered by them